llvm/test/CodeGen/DirectX/countbits.ll

```diff
@@ -0,0 +1,28 @@
+; RUN: opt -S -dxil-op-lower -mtriple=dxil-pc-shadermodel6.3-library %s | FileCheck %s
```

Comment:
so I don't have to clean this up later can you change this to 
```suggestion
; RUN: opt -S -scalarizer -dxil-op-lower -mtriple=dxil-pc-shadermodel6.3-library %s | FileCheck %s
```
And then add a vector test.